modules/png-extraction/config.json

```diff
@@ -5,6 +5,7 @@
 	"SplitIntoChunks": 1,
 	"PrintImages": true,
 	"CommonHeadersOnly": false,
+	"PublicHeadersOnly": true,
```

Comment:
Good

Reply:
Make sure to update the README (https://github.com/Emory-HITI/Niffler/blob/dev/modules/png-extraction/README.md) too, to introduce this new property and how to use this.


Add your changes as:


### Print the Images or Limit the Extraction to Include only the Common DICOM Attributes

The below three fields can be left unmodified for most executions. The default values are included below for these boolean properties.

* *PrintImages*: Do you want to print the images from these dicom files? Default is _true_.

* *CommonHeadersOnly*: Do you want the resulting dataframe csv to contain only the common headers? Finds if less than 10% of the rows are missing this column field. To extract all the headers, default is set as _false_.

* *PublicHeadersOnly*: .......add description.........

Reply:
Ok , I do it.